TestLibrary/TestPatch.cs

```diff
@@ -0,0 +1,23 @@
+﻿using HarmonyLib;
+using System.Collections.Generic;
+using System.Reflection;
+
+namespace TestLibrary;
+
+public static class TestPatch
+{
+	private static void Method() { }
+
+	private static IEnumerable<CodeInstruction> EmptyTranspiler(IEnumerable<CodeInstruction> instructions) => instructions;
+
+
+	public static void TestPatching(out Assembly original, out Assembly patched)
+	{
+		original = Assembly.GetExecutingAssembly();
+
+		var instance = new Harmony("test");
+		instance.Patch(SymbolExtensions.GetMethodInfo(() => Method()), transpiler: SymbolExtensions.GetMethodInfo(() => EmptyTranspiler(null)));
+
+		patched = Assembly.GetExecutingAssembly();
+	}
```

Comment:
Shouldn't we also test the case where the original method calls Assembly.GetExecutingAssembly?

Reply:
Can't believe I forgot that. Fixed in the latest commit